### PR TITLE
Separate user DB per component

### PR DESCRIPTION
**Pull request by jasiu001**
**Description**

Changes proposed in this pull request:
- Separation of all database clients for individual components

### CODE REVIEW

File: resources/compass/charts/director/templates/deployment.yaml [hunk 1]
```diff
@@ -61,7 +61,7 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: compass-postgresql
-                  key: postgresql-directorDatabaseName
+                  key: postgresql-director-name
```
Comment (aszecowka) [inline resources/compass/charts/director/templates/deployment.yaml]: maybe better will be: `postgresql-director-db-name`? The same in other places when we define db name

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 2]
```diff
@@ -18,11 +18,15 @@ To install Compass with GCP managed Postgres database, set the **database.embedd
 | --- | --- | --- | --- |
 | `global.database.managedGCP.serviceAccountKey` | Specifies base64 encoded the key for GCP Service Account mentioned in prerequisites. | base64 encoded string | "" |
 | `global.database.managedGCP.instanceConnectionName` | Specifies instance connection name to GCP PostgreSQL database | string | "" |
-| `global.database.managedGCP.dbUser` | Specifies database username | string | "" |
-| `global.database.managedGCP.dbPassword` | Specifies password for database user | string | "" |
-| `global.database.managedGCP.directorDBName` | Specifies Director database name | string | "" |
-| `global.database.managedGCP.provisionerDBName` | Specifies Provisioner database name | string | "" |
-| `global.database.managedGCP.brokerDBName` | Specifies Kyma Environment Broker database name | string | "" |
+| `global.database.managedGCP.director.user` | Specifies Director database username | string | "" |
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
| `global.database.managedGCP.director.user` | Specifies the database username in the Director. | string | "" |
```

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 3]
```diff
@@ -18,11 +18,15 @@ To install Compass with GCP managed Postgres database, set the **database.embedd
...
+| `global.database.managedGCP.director.password` | Specifies Director password for database user | string | "" |
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
| `global.database.managedGCP.director.password` | Specifies the password to the user's database in the Director. | string | "" |
```

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 4]
```diff
@@ -18,11 +18,15 @@ To install Compass with GCP managed Postgres database, set the **database.embedd
...
+| `global.database.managedGCP.director.name` | Specifies Director database name | string | "" |
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
| `global.database.managedGCP.director.name` | Specifies the database name in the Director. | string | "" |
```

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 5]
```diff
@@ -18,11 +18,15 @@ To install Compass with GCP managed Postgres database, set the **database.embedd
...
+| `global.database.managedGCP.provisioner.user` | Specifies Provisioner database username | string | "" |
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
| `global.database.managedGCP.provisioner.user` | Specifies the database username in the Runtime Provisioner. | string | "" |
```

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 6]
```diff
@@ -18,11 +18,15 @@ To install Compass with GCP managed Postgres database, set the **database.embedd
...
+| `global.database.managedGCP.provisioner.password` | Specifies Provisioner password for database user | string | "" |
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
| `global.database.managedGCP.provisioner.password` | Specifies the password to the user's database in the Runtime Provisioner. | string | "" |
```

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 7]
```diff
@@ -18,11 +18,15 @@ To install Compass with GCP managed Postgres database, set the **database.embedd
...
+| `global.database.managedGCP.provisioner.name` | Specifies Provisioner database name | string | "" |
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
| `global.database.managedGCP.provisioner.name` | Specifies the database name in the Runtime Provisioner. | string | "" |
```

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 8]
```diff
@@ -18,11 +18,15 @@ To install Compass with GCP managed Postgres database, set the **database.embedd
...
+| `global.database.managedGCP.broker.user` | Specifies Kyma Environment Broker database username | string | "" |
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
| `global.database.managedGCP.broker.user` | Specifies the database username in Kyma Environment Broker. | string | "" |
```

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 9]
```diff
@@ -18,11 +18,15 @@ To install Compass with GCP managed Postgres database, set the **database.embedd
...
+| `global.database.managedGCP.broker.password` | Specifies Kyma Environment Broker password for database user | string | "" |
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
| `global.database.managedGCP.broker.password` | Specifies the password to the user's database in Kyma Environment Broker. | string | "" |
```

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 10]
```diff
@@ -18,11 +18,15 @@ To install Compass with GCP managed Postgres database, set the **database.embedd
...
+| `global.database.managedGCP.broker.name` | Specifies Kyma Environment Broker database name | string | "" |
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
| `global.database.managedGCP.broker.name` | Specifies the database name in Kyma Environment Broker.| string | "" |
```

File: resources/compass/configure-managed-gcp-postgresql.md [hunk 11]
```diff
@@ -36,7 +40,7 @@ Than look for `Instance connection name` box inside `Connect to this instance` s
 
 ![Instance connection String](./assets/instance-connection-string.png)
 
-For `dbUser` and `dbPassword` values, use one of accounts from `USERS` tab. The `dbName` value is name of database which you want to use. The available names can be found inside `DATABASES` tab.
+For `director.user`, `provisioner.user`, `broker.user` and `director.password`, `provisioner.password`, `broker.password` values, use one of accounts from `USERS` tab. The `director.name`, `provisioner.name`, `broker.name` values is name of database which you want to use. The available names can be found inside `DATABASES` tab.
```
Comment (klaudiagrz) [inline resources/compass/configure-managed-gcp-postgresql.md]:
```suggestion
For the `director.user`, `provisioner.user`, `broker.user`, `director.password`, `provisioner.password`, and `broker.password` values, use one of the accounts from the **USERS** tab. The `director.name`, `provisioner.name`, and `broker.name` values are the names of available databases. For the list of available names, see the **DATABASES** tab.
```

**Pull request merged**